tests/app/config/database.php

```diff
@@ -6,9 +6,9 @@
         'mysql' => [
             'driver'    => 'mysql',
             'host'      => (getenv('DB_HOST')) ?: '127.0.0.1',
-            'database'  => (getenv('DB_NAME')) ?: 'gzero-tests',
+            'database'  => (getenv('DB_NAME')) ?: 'gztest',
```

Comment:
We shouldn't change default database settings, because jenkins is using them.
If you want to use you own settings, you need to ignore this file when committing.